src/modules/map/components/cesium/CesiumMap.vue

```diff
@@ -552,24 +552,23 @@ export default {
                 coordinates = proj4(this.projection.epsg, WEBMERCATOR.epsg, featureCoords)
             }
             this.click({
-                clickInfo: new ClickInfo(
+                clickInfo: new ClickInfo({
                     coordinates,
```

Comment:
For the feature selection bug in 3d : the field in `ClickInfo` is called `coordinate` and not `coordinates`, which means this can't translate it to `coordinate: value`.

Either we change this to `coordinate: coordinates` or we change `coordinates` to `coordinate` everywhere in this file.